app/src/main/res/authors.txt

```diff
@@ -84,6 +84,16 @@ kerb_height_flush_original.jpg CC0               Mateusz Konieczny (...)
 kerb_height_lowered_ramp.jpg   CC0               https://commons.wikimedia.org/wiki/File:Pram_Ramp.jpg (Michael3)
 kerb_height_no.jpg             Public Domain     https://commons.wikimedia.org/wiki/File:Ground_frost_damages.JPG (SeppVei)
 
+memorial_type_statue.jpg       CC-BY-SA 4.0      Rabax63 https://commons.wikimedia.org/wiki/File:Garibaldi_Monument.jpg
+memorial_type_bust.jpg         CC0               Jebulon https://commons.wikimedia.org/wiki/File:Aristotelis_Valaoritis_bust_National_Garden_Athens,_Greece.jpg
+memorial_type_plaque.jpg       Public Domain     Geotek https://commons.wikimedia.org/wiki/File:Bleaklow_Plaque.jpg
```

Comment:
Maybe contrast can be tweaked a bit for `memorial_type_plaque`?

Reply:
https://github.com/streetcomplete/StreetComplete/pull/4102/commits/f5a9af05b214c11a5888d97f34b80cf5f64e1eb9

Reply:
![screen05](https://user-images.githubusercontent.com/899988/173232680-a04b809f-c955-417f-a7c6-7cbc81057cc2.png)